src/__tests__/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { checkReactElement, isEmpty, getStylePropAsRecord } from '../utils';
+
+describe('checkReactElement', () => {
+  test('it does not throw an error for valid native primitives', () => {
+    expect(() => {
+      // @ts-ignore how to type it?
+      checkReactElement({ type: 'Text' }, () => {}, null);
```

Comment:
What is the TS error here?

Reply:
{ type: 'Text} is not a valid ReactTestInstance so tsc complains

Reply:
Yeah, properly mocking `ReactTestInstance` interface would be much more complex, without adding significant benefit beside removing // @ts comments